L/LBFGSB/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LBFGSB"
+version = v"3.0.0"
+
+# Collection of sources required to build LBFGSB
+sources = [
+    "http://users.iems.northwestern.edu/~nocedal/Software/Lbfgsb.3.0.tar.gz" =>
+    "f5b9a1c8c30ff6bcc8df9b5d5738145f4cbe4c7eadec629220e808dcf0e54720",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Lbfgsb.*/
+mkdir -p "${libdir}"
+FFLAGS="-O3 -fPIC -shared -Wall -fbounds-check -Wno-uninitialized"
+${FC} ${LDFLAGS} ${FFLAGS} lbfgsb.f linpack.f blas.f timer.f -o "${libdir}/liblbfgsb.${dlext}"
+install_license License.txt
```

Comment:
We have to manually install the license file because the source tarball has a hidden file in the top-level, in addition to the source directory:
```
sandbox:${WORKSPACE}/srcdir/Lbfgsb.3.0 # ls -a ${WORKSPACE}/srcdir/
.             ..            ._Lbfgsb.3.0  Lbfgsb.3.0
sandbox:${WORKSPACE}/srcdir/Lbfgsb.3.0 # file ${WORKSPACE}/srcdir/._Lbfgsb.3.0 
/workspace/srcdir/._Lbfgsb.3.0: AppleDouble encoded Macintosh file
```
:expressionless:

Reply:
> Note that `${FC}` is already set in the build environment to the right value (currently `gfortran` everywhere, but it may change in the future), and the variable `${dlext}` already holds the extension for the shared library. The library should be installed to `${libdir}`, not `${prefix}/lib`. See [BinaryBuilder documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1).

Looks like the documentation has been greatly improved since the new 0.2.0 release, thanks for your nice work on this!